### ISSUE TITLE
chore: update to iOS AppNexusSDK version 9.0.0

### ISSUE THREAD

**Issue by thekorn** [thekorn/xandr#145]
This has already bee tried in version 0.2.9 - however this resulted in failed validation on package submission

<img width="783" alt="Bildschirmfoto 2024-08-15 um 15 36 00" src="https://github.com/user-attachments/assets/03c902ca-3387-4233-bbd4-f6e2cff5c0e5">

```
Unexpected CFBundleExecutable Key. The bundle at 'Payload\/Runner.app\/ANSDKResources.bundle' does not contain a bundle executable. If this bundle intentionally does not contain an executable, consider removing the CFBundleExecutable key from its Info.plist and using a CFBundlePackageType of BNDL. If this bundle is part of a third-party framework, consider contacting the developer of the framework for an update to address this issue. (ID: 4d103e4e-eb6d-47e8-838c-a2933796e320)
```

An upstream ticket has been created, waiting for their feedback/fix here: https://help.xandr.com/s/case/5006S00002VMmAnQAL/ios-sdk-validation-error-when-using-version-900

**Comment by thekorn** [comment]
@ashcherbonos FYI, I created this ticket to track the efforts to update the iOS sdk to version 9.0.0 (released to https://github.com/thekorn/xandr/pull/144)

**Comment by thekorn** [comment]
this has been fixed with latest release of this package 0.2.13
